src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx

```diff
@@ -337,13 +354,13 @@ Supported databases and libraries:
   </tbody>
 </table>
 
-## Other APM software [#other-monitoring-software]
+## Other third-party monitoring software [#other-monitoring-software]
 
-If your application uses other application performance monitoring (APM) software besides our agent, we cannot guarantee that our agent will work correctly and we cannot offer technical support. For more information, see [Errors when using other monitoring software](/docs/apm/new-relic-apm/troubleshooting/errors-while-using-new-relic-apm-alongside-other-apm-software).
+If your application uses any additional monitoring software, we cannot guarantee that our agent will work correctly and cannot offer technical support. For more information, see [Errors when using other monitoring software](/docs/apm/new-relic-apm/troubleshooting/errors-while-using-new-relic-apm-alongside-other-apm-software).
 
 ## Instance details
```

Comment:
@ally-sassman, this looks like it's related to the main database section above. You might check with the SME to see if there's a way to combine these database instance details with the other database section. Or, consider at least moving this right below the database section.